doc/classes/Node.xml

```diff
@@ -275,6 +275,7 @@
 				Negative indices access the children from the last one.
 				If [param include_internal] is [code]false[/code], internal children are skipped (see [code]internal[/code] parameter in [method add_child]).
 				To access a child node via its name, use [method get_node].
+				If no child exists at the index, [code]null[/code] is returned and an error is logged.
```

Comment:
This should instead be added to the first paragraph. And it could also benefit from slight rephrasing. Something like

```
Returns a child node by its index (see also [method get_child_count]). If no child exists at the given index, this method returns [code]null[/code] and an error is logged. This method is often used for iterating all children of a node.
```

Reply:
Updated.